pkg/omf/cli/omf.theme.fish

```diff
@@ -1,6 +1,64 @@
-function omf.theme
-  if not test -e $OMF_CONFIG/themes/$argv[1]
-    omf.install --theme $argv[1]
+function omf.theme -a target_theme
+  set -l current_theme (cat $OMF_CONFIG/theme)
+
+  test "$target_theme" = "$current_theme"; and begin; echo Already at; return 0; end
+
+  if not contains -- "$target_theme" (omf.list_installed_themes)
+    if not omf.install --theme "$target_theme"
+      return 1
+    end
   end
-  echo "$argv[1]" > $OMF_CONFIG/theme
+
+  set -l current_theme_path
+  set -l target_theme_path
+
+  for path in {$OMF_CONFIG,$OMF_PATH}/themes
```

Comment:
don't we want to reverse these, so `$OMF_CONFIG` wins over `$OMF_PATH` if they both have the same theme name?